src/index.html

```diff
@@ -8,5 +8,40 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <main class="calendar">
+      <div class="calendar__container calendar__container--start-day--wed calendar__container--month-length--30">
```

Comment:
The logic is reversed.
Outer block is the container, inner block is calendar.
Make it a separate block, thus reducing length of line 12.